src/main/java/killrvideo/configuration/DseConfiguration.java

```diff
@@ -57,6 +66,12 @@
    
     @Value("#{environment.KILLRVIDEO_DSE_PASSWORD}")
     public Optional < String > dsePassword;
+
+    @Value("#{environment.KILLRVIDEO_ENABLE_SSL}")
+    public Optional < Boolean > dseEnableSSL;
+
+    @Value("${killrvideo.ssl.CACertFileLocation: 'cassandra.cert'}")
```

Comment:
I propose to use the key `killrvideo.cassandra.ssl.CACertFileLocation` in the `application.yaml` to identify that the SSL connectivity is associated with DSE.

Reply:
Good point, done